ts/components/messages/MessageAgenda.tsx

```diff
@@ -364,29 +434,78 @@ class MessageAgenda extends React.PureComponent<Props, State> {
       refreshing,
       onContentSizeChange
     } = this.props;
+
+    const { onMoreDataRequest } = this.props;
+
+    const marginTopAnim = this.overScrollAnim.interpolate({
+      inputRange: [0, HEADER_SCROLL_DISTANCE],
+      outputRange: [0, HEADER_SCROLL_DISTANCE],
+      extrapolate: "clamp"
+    });
+
+    const marginTopList = sections.length === 0 ? 0 : -HEADER_SCROLL_DISTANCE;
+
     return (
-      <SectionList
-        ref={this.sectionListRef}
-        ListEmptyComponent={ListEmptyComponent}
-        ListHeaderComponent={this.renderListHeader}
-        renderSectionHeader={this.renderSectionHeader}
-        renderItem={this.renderItem}
-        ItemSeparatorComponent={ItemSeparatorComponent}
-        sections={sections}
-        extraData={{ servicesById, paymentsByRptId }}
-        refreshing={refreshing}
-        onContentSizeChange={onContentSizeChange}
-        stickySectionHeadersEnabled={true}
-        keyExtractor={keyExtractor}
-        getItemLayout={this.getItemLayout}
-        bounces={false}
-      />
+      <View style={styles.fill}>
+        <Animated.View
```

Comment:
Are this view really animated? The style is always `styles.button`? Am i missing something?

Reply:
> Are this view really animated? The style is always `styles.button`? Am i missing something?

It isn't really animated, but, if the button not is child of an animated-view the button is not showed! I try to inspected this element better...